task_01.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A function to convert Fahrenheit temperature to Celcius."""
+
+
+import decimal
+
+import task_02
+def fahrenheit_to_celcius(degrees):
```

Comment:
The error `'module' object has no attribute 'fahrenheit_to_celsius'` is indicative of a misspelling since it can't find a function with that name.